doc/SPO-MANUAL-MIGRATION-GUIDE.md

```diff
@@ -0,0 +1,564 @@
+  oc get rawselinuxprofile $name -n openshift-security-profiles -o json | jq '.' > \
+    "$BACKUP_DIR/profiles/rawselinuxprofiles-openshift-security-profiles-${name}.json"
+  echo "Backed up rawselinuxprofile: $name"
+done
+```
+
+**Actual output:**
+```
+Backed up seccompprofile: profile-block-all
+Backed up selinuxprofile: errorlogger-selinuxd-test
+Backed up rawselinuxprofile: errorlogger
+```
+
+### Create Backup Summary
+```bash
+echo "=== Backup Summary ===" > $BACKUP_DIR/backup-summary.txt
+echo "Total files backed up: $(find $BACKUP_DIR -name "*.json" | wc -l)" >> $BACKUP_DIR/backup-summary.txt
+echo "" >> $BACKUP_DIR/backup-summary.txt
+echo "SPOD configuration: $([ -f $BACKUP_DIR/spod-configuration.json ] && echo "Yes" || echo "No")" >> $BACKUP_DIR/backup-summary.txt
+echo "" >> $BACKUP_DIR/backup-summary.txt
+echo "Profiles backed up:" >> $BACKUP_DIR/backup-summary.txt
+find $BACKUP_DIR/profiles -name "*.json" -exec basename {} \; | sort >> $BACKUP_DIR/backup-summary.txt
+echo "" >> $BACKUP_DIR/backup-summary.txt
+echo "Profile bindings backed up:" >> $BACKUP_DIR/backup-summary.txt
+find $BACKUP_DIR/bindings-recordings -name "*.json" -exec basename {} \; | sort >> $BACKUP_DIR/backup-summary.txt
+
+cat $BACKUP_DIR/backup-summary.txt
+```
+
+**Actual output:**
+```
+=== Backup Summary ===
+Total files backed up: 6
+
+SPOD configuration: Yes
+
+Profiles backed up:
+rawselinuxprofiles-openshift-security-profiles-errorlogger.json
+seccompprofiles-openshift-security-profiles-profile-block-all.json
+selinuxprofiles-openshift-security-profiles-errorlogger-selinuxd-test.json
+
+Profile bindings backed up:
+profilebindings-openshift-security-profiles-profile-binding-seccomp.json
+profilebindings-openshift-security-profiles-profile-binding-selinuxd.json
+```
+
+## Step 4: Delete SPO Objects in Order
+
+**IMPORTANT**: Delete profile bindings first to prevent the operator from automatically reapplying profiles to workloads.
+
+### Delete Profile Bindings First
+```bash
+# List and delete all profile bindings
+oc get profilebindings -n openshift-security-profiles
+oc delete profilebindings --all -n openshift-security-profiles
+```
+
+**Actual command and output:**
+```bash
+oc delete profilebinding profile-binding-seccomp profile-binding-selinuxd -n openshift-security-profiles
+```
+```
+profilebinding.security-profiles-operator.x-k8s.io "profile-binding-seccomp" deleted
+profilebinding.security-profiles-operator.x-k8s.io "profile-binding-selinuxd" deleted
+```
+
+### Update Workloads to Remove Profile References
+
+Next, update all workloads to remove references to SPO profiles. You must either recreate pods without these profiles, or update deployments, statefulset, and other workloads to remove the profile references from their pod templates.
+
+
+### Delete Security Profiles
+```bash
+# Delete all security profiles
+oc delete seccompprofiles,selinuxprofiles,rawselinuxprofiles --all -n openshift-security-profiles
```

Comment:
same as above, the seccompprofiles, selinuxprofiles and rawselinuxprofiles can be created in all namespaces.